lib/elixir_google_scraper_web/controllers/api/v1/keyword_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule ElixirGoogleScraperWeb.Api.V1.KeywordController do
+  use ElixirGoogleScraperWeb, :controller
+
+  alias ElixirGoogleScraper.Scraper.Keywords
+  alias ElixirGoogleScraperWeb.V1.ErrorView
+
+  plug(ExOauth2Provider.Plug.VerifyHeader, otp_app: :elixir_google_scraper, realm: "Bearer")
+
+  plug(ExOauth2Provider.Plug.EnsureAuthenticated,
+    handler: ElixirGoogleScraperWeb.Api.ErrorHandler
+  )
+
+  def create(conn, %{"file" => %Plug.Upload{} = file}) do
+    token = ExOauth2Provider.Plug.current_access_token(conn)
```

Comment:
Should we move this logic to "plug" to get the current resource owner and store it in `assigns`? Then we include the "plug" in the router.

So in the controller we can use it by `conn.assigns.user`

Reply:
This is a good idea!, assigned the user to the `conn` and moved the code into a sharable place. Thanks